tests/lib/Grpc.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Store } from '@sapphire/pieces';
+
+const { grpc } = Store.injectedContext;
+
+describe('Grpc', () => {
```

Comment:
I feel like it's probably a smart idea to wipe the db before every run again, as we do in Skyra.Grpc tests - otherwise tests depend on a fresh database, which might not be the case for some development environments

Reply:
Not possible, you didn't add something to nuke the entire DB.